0-ad-history-mod-package/simulation/templates/units/han/infantry_spearman_b.xml

```diff
@@ -10,6 +10,11 @@
     <SelectionGroupName>units/han/infantry_spearman_b</SelectionGroupName>
     <SpecificName>Máo Bīng</SpecificName>
     <Icon>units/han/infantry_spearman.png</Icon>
+    <History>
+        In the Han dynasty, the typical combination of spear and shield was rivaled by other, more advanced weaponry. However, ordinary spearmen were still very common. This was because shields and spears required low effort and expertise to manufacture. And fighting with a shield and spear was quite easy to learn, which was important because Han armies largely consisted of militia.
```

Comment:
```suggestion
        In the Han dynasty, the typical combination of spear and shield was rivaled by other, more advanced weaponry. However, ordinary spearmen were still very common. This was because shields and spears required little effort and expertise to manufacture. And fighting with a shield and spear was easy to learn, which was important because Han armies largely consisted of militia.
```